acceptance/bundle/state/same_serial/script

```diff
@@ -0,0 +1,6 @@
+title 'If serial is the same, refuse to proceed'
+mkdir -p .databricks/bundle/default/terraform
+echo '{"version": 4, "serial": 20}' > .databricks/bundle/default/terraform/terraform.tfstate
+echo '{"serial": 20}' > .databricks/bundle/default/resources.json
```

Comment:
No lineage?

We could also assert that lineage is set, otherwise we can't compare the serials.